vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -996,9 +1010,38 @@ def grouped_topk(
     if renormalize:
         topk_weights = topk_weights / topk_weights.sum(dim=-1, keepdim=True)
 
+    topk_weights = topk_weights * routed_scaling_factor
```

Comment:
nit/future work: maybe we should default routed_scaling_factor to None so we can skip the multiply if it isn't set